### DISCUSSION TITLE
Is it possible to wrap mdx files with Providers from a global configuration (v7)

### DISCUSSION THREAD

**Discussion by vmichailidisdev** [storybookjs/storybook#24457] [Help]
Hello there,

Started following this guide to integrate Mui with storybook https://storybook.js.org/blog/material-ui-in-storybook/

The following code is used for wrapping all stories with the Mui provider

`preview.js`
```jsx
{
  decorators: [
    (Story) => (
      <>
        <ThemeProvider theme={baseTheme}>
          <CssBaseline />
          <Story />
        </ThemeProvider>
      </>
    ),
  ]
}
```

However, this configuration does not wrap .mdx files. So when Mui components are used in .mdx files, errors occur.

Does a configuration exist that covers both stories and .mdx files?

This configuration did not work:

`preview.js`
```jsx
{
  parameters: {
    docs: {
      container: ({ children, context }) => (
        <DocsContainer context={context}>
          <ThemeProvider theme={baseTheme}>
            <CssBaseline />
            {children}
          </ThemeProvider>
        </DocsContainer>
      ),
    },
  }
}
```

**Comment by HaleyWardo** [comment]
@vmichailidisdev any luck on this solution? I am hoping to achieve the same thing.

**Reply by vmichailidisdev** [reply]
Hello, sorry for the late response. Unfortunately no.

**Comment by jbenning97** [comment]
Following this thread, same thing here.
